src/java/com/sarality/app/view/action/TriggerType.java

```diff
@@ -8,6 +8,10 @@
 public enum TriggerType {
   CLICK,
   CLICK_LIST_ITEM,
+  DIALOG_CANCEL,
+  DIALOG_POSITIVE,
+  DIALOG_NEGATIVE,
+  DIALOG_NEUTRAL,
```

Comment:
can we have comments also for this please. Neutral is not as intuitive as the others